### ISSUE TITLE
New DND5e charsheets navigation is broken

### ISSUE THREAD

**Issue by fris42** [mouse0270/window-tabs#7]
Hello!

I had to turn 'window tabs' mod off because it broke DnD5e official system Character Sheet navigation (now they use tabs on the right side of the sheet. spells, inventory, affects, etc...).

upd: I am talking about this system https://github.com/foundryvtt/dnd5e/ and versions 3+

**Comment by abbeyroad7** [comment]
Also disables the edit option in the top left. Sadly had to disable module as well.

**Comment by jendives2000** [comment]
Indeed, this is still the case on the V11 - 315 stable and DnD system 3.1.2. 
![image](https://github.com/mouse0270/window-tabs/assets/94803007/752462d8-8cd1-48b7-982d-1d115aa6dafe)
